pkg/platform/runtime/alternative.go

```diff
@@ -216,6 +217,18 @@ func (ai *AlternativeInstall) PreInstall() error {
 				return locale.WrapError(err, "err_rm_artf", "", "Could not remove old package file at {{.V0}}.", file)
 			}
 		}
+
+		for _, dir := range v.Dirs {
```

Comment:
We probably want to iterate over the directories in reverse alphabetical order, such that you look `a/b/c` before `a/b`.  Otherwise you might only remove `the first one, but not the latter one.

Reply:
Nice! `IsEmptyDirs` was to address this but sorting also works.